crates/dbsp/src/trace/persistent/trace.rs

```diff
@@ -389,6 +370,9 @@ where
                                 .count()
                                 != 0
                         });
+                    } else {
+                        //tws.sort_unstable_by(|(t1, _), (t2, _)| t1.cmp(t2));
```

Comment:
I don't know (anymore) why this line was there commented previously, maybe time to get rid of it then :)?

Reply:
Done, thanks.